manager/ping/ping_test.go

```diff
@@ -0,0 +1 @@
+package ping
```

Comment:
Remove this file if there's no test (it doesn't really seem wroth writing a test for two one-line functions).